olm-catalog/serverless-operator/1.7.0/serverless-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -57,6 +57,15 @@ metadata:
               }
             }
           }
+        },
+        {
+          "apiVersion": "operator.knative.dev/v1alpha1",
+          "kind": "KnativeEventing",
+          "metadata": {
+            "name": "knative-eventing"
+          },
+          "spec": {
+          }
```

Comment:
Yes, there's nothing in the `KnativeEventing` spec, other than the image override stuff. But we don't do the image override in that ALM example block. We want the users to provide empty spec and then our serverless operator will modify the CR with overrides we like and then it will let the upstream operator do its job.